lms/templates/calculator/toggle_calculator.html

```diff
@@ -0,0 +1,133 @@
+<%! from django.utils.translation import ugettext as _ %>
+<%! from django.core.urlresolvers import reverse %>
+
+<div class="calc-main">
+    <a title="${_('Open Calculator')}" href="#" role="button" aria-controls="calculator_wrapper" aria-expanded="false" class="calc">${_("Open Calculator")}</a>
+
+    <div id="calculator_wrapper">
+      <form id="calculator">
+        <div class="input-wrapper">
+          <input type="text" id="calculator_input" title="${_('Calculator Input Field')}" tabindex="-1" />
+
+          <div class="help-wrapper">
+            <p class="sr" id="hint-instructions">${_('Use the arrow keys to navigate the tips or use the tab key to return to the calculator')}</p>
+
+            <a id="calculator_hint" href="#" role="button" aria-haspopup="true" tabindex="-1" aria-describedby="hint-instructions">${_("Hints")}</a>
+
+            <ul id="calculator_input_help" class="help" aria-activedescendant="hint-moreinfo" role="tooltip" aria-hidden="true">
+              <li class="hint-item" id="hint-moreinfo" tabindex="-1">
+                <p><span class="bold">
+                  ${_("For detailed information, see {link_mathexpressions_start}Entering Mathematical and Scientific Expressions{link_mathexpressions_end} in the {link_studentguide_start}edX Guide for Students{link_studentguide_end}.format(
+              link_mathexpressions_start='<a href="http://edx-guide-for-students.readthedocs.org/en/latest/SFD_mathformatting.html">', link_mathexpressions_end="</a>, link_studentguide_start='<a href="hhttp://edx-guide-for-students.readthedocs.org/en/latest/index.html">', link_studentguide_end="</a>"
```

Comment:
Also, are we using underscores for IDs and dashes for classnames? Or should they all be dashes?